Sources/SwiftQueue/Constraint.swift

```diff
@@ -47,6 +47,8 @@ public enum RetryConstraint {
     case cancel
     /// Exponential back-off
     case exponential(initial: TimeInterval)
+    /// Exponential back-off with max delay
+    case exponentialWithMaxDelay(initial: TimeInterval, maxDelay: TimeInterval)
```

Comment:
Can name it `exponentialWithLimit` instead ?

Reply:
> Can name it `exponentialWithLimit` instead ?

All done